src/libraries/tests.proj

```diff
@@ -762,7 +762,8 @@
   <ItemGroup Condition="'$(ArchiveTests)' == 'true' and '$(TargetFrameworkIdentifier)' == '.NETCoreApp' and '$(IsXUnitLogCheckerSupported)' == 'true'">
     <ProjectReference
       Include="$(RepoRoot)src\tests\Common\XUnitLogChecker\XUnitLogChecker.csproj"
-      AdditionalProperties="%(AdditionalProperties);Configuration=Release;OutDir=$(XUnitLogCheckerLibrariesOutDir)" />
+      Targets="Publish"
+      AdditionalProperties="%(AdditionalProperties);_IsPublishing=true;Configuration=Release;PublishDir=$(XUnitLogCheckerLibrariesOutDir)" />
```

Comment:
For my own education, why is `_IsPublishing` being passed? Why isn't int sufficient to specify `Targets="Publish"` here?

Reply:
This is tricky. In short, `_IsPublishing` is required for publishing, but not set when you're using `msbuild /t:publish`, only when you're doing `dotnet publish`.